src/transformers/modeling_funnel.py

```diff
@@ -187,16 +187,16 @@ def __init__(self, config):
         # dividide.
         self.pooling_mult = None
 
-    def init_attention_inputs(self, input_embeds, attention_mask=None, token_type_ids=None):
+    def init_attention_inputs(self, inputs_embeds, attention_mask=None, token_type_ids=None):
```

Comment:
@sgugger - I renamed the variable here because it's always called `inputs_embeds` in the lib and funnel was the only model that had `input_embeds` at some point. It's just a nit because it's not exposed to the user, but I think it's better for consistency.